src/gfx/egui/src/input.rs

```diff
@@ -237,6 +237,7 @@ pub fn translate_raw_input(input: &egui_Input) -> Result<egui::RawInput, Transla
         screen_rect: Some(input.screenRect.into()),
         pixels_per_point: Some(input.pixelsPerPoint),
         max_texture_side: None,
+        has_focus: true, // FIXME
```

Comment:
@guusw need to check whether it's correct to set `has_focus` to `true` all the time. We might need some other logic when we have multiple UIs in the same GFX scene.

Note: it is a new field on this struct introduced by egui in this release.

Reply:
Good to keep true for now.
We'll filter input events once we have multiple instances but we can set this too then.